packages/paste-website/src/components/breadcrumb/index.tsx

```diff
@@ -18,7 +18,7 @@ const Breadcrumb: React.FC = ({children}) => {
       <Box
         as="ol"
         display="flex"
-        margin="space0"
+        margin="space10"
```

Comment:
Do we need to revert all these changes?

Reply:
Yup, they are in their own test case commit. I'll bin them once we're happy with proof of working